assets/add_new_asset.md

```diff
@@ -283,6 +285,22 @@ To do changes in the assets repository, you need to create a personal copy calle
 Once the changes are prepared inside the fork, you need to create a _pull request_ to the main repository.
 Upon review the maintainers will accept your pull request, and the changes will be incorporated.
 
+#### Adding files using Assets App
```

Comment:
can you delete `Adding files using command-line` devs usually would know how to do it, and the rest can use assets.trustwallet.com